Source/WebCore/css/typedom/transform/CSSPerspective.cpp

```diff
@@ -69,6 +73,27 @@ ExceptionOr<Ref<CSSPerspective>> CSSPerspective::create(CSSPerspectiveValue leng
     return adoptRef(*new CSSPerspective(checkedLength.releaseReturnValue()));
 }
 
+ExceptionOr<Ref<CSSPerspective>> CSSPerspective::create(CSSFunctionValue& cssFunctionValue)
+{
+    if (cssFunctionValue.name() != CSSValuePerspective) {
+        ASSERT_NOT_REACHED();
+        return CSSPerspective::create("none"_s);
+    }
+
+    ASSERT(cssFunctionValue.size() == 1 && cssFunctionValue.item(0));
```

Comment:
Let's make this a little more bullet proof with a check, ASSERT_NOT_REACHED, return an exception.

Reply:
Addressed in 6b8ab023c1ea.